app/api/events.py

```diff
@@ -63,6 +63,15 @@ class Meta:
                            schema='SponsorSchema',
                            many=True,
                            type_='sponsor')
+    notification = Relationship(
+        attribute='notification',
+        self_view='v1.event_notification',
```

Comment:
I think you haven't created relationship end point for ```event_notification```. Please let me know if I missed it.